tekton/tasks/qeb-hwt-task.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: qeb-hwt
+  namespace: default
+spec:
+  inputs:
+    resources:
+      - name: git-repo # target path is /workspace otherwise targetPath: specific needs to be included
+        resourceSpec:
+          type: git
+          description: "The input is code from a git repository"
+
+  outputs:
+    resources:
+      - name: justification
+        type: string
+        description: "Justification from adviser run"
+  steps:
+    - name: thamos-advise
+      image: "quay.io/pacospace/thamos:v0.8.1-dev"  # image need to contain python
+      script: |
+        #!/usr/bin/env python3
```

Comment:
if we want to run this script, maybe we can put it directly inside the container image. WDYT?

Reply:
But this script might change in the future, what is better to do for maintainability? We put this script in .py file and we create the image each time with that script inside?

Reply:
Possible having a script .py file and creating an image on each change, feels like a better approach to me

Reply:
Feel free to send a commit for that :)